### DISCUSSION TITLE
I can’t create tables in a local instance of supabase

### DISCUSSION THREAD

**Discussion by bl-aire** [supabase/supabase#23405] [Questions]
I setup a local instance of supabase using the instructions in DEVELOPER.md 

- Forked the repo
- Cloned the forked repo
- Ran  _cd supabase_
- Ran _npm install_
- Copied the example .env.local.example to .env.local by running _cp apps/www/.env.local.example apps/www/.env.local_
- Ran _npm run dev_
- Opened http://localhost:8082 on my browser 

Here’s what it looks like: 

![Image 29-04-2024 at 12 06](https://github.com/supabase/supabase/assets/84882370/c1264a79-690c-4005-8bb4-76701a36f115)

And, I get these errors on my browser’s console on clicking save:

@encima 

![Image 29-04-2024 at 13 57](https://github.com/supabase/supabase/assets/84882370/8d79fe44-b8e1-45c0-b3f2-0457723914dc)

**Comment by encima** [comment]
Hi @bl-aire 
thanks for opening and welcome to Supabase!

Can you confirm you have the whole stack running? The steps you outline are to run the frontend app only.

the easiest way to run Supabase locally is:
- `npx supabase@latest init`
- `npx supabase@latest start`

If you need to do it from the repo, you can follow the steps [here](https://supabase.com/docs/guides/self-hosting/docker)

**Reply by bl-aire** [reply]
> Hi @bl-aire 
> 
> thanks for opening and welcome to Supabase!
> 
> 
> 
> Can you confirm you have the whole stack running? The steps you outline are to run the frontend app only.
> 
> 
> 
> the easiest way to run Supabase locally is:
> 
> - `npx supabase@latest init`
> 
> - `npx supabase@latest start`
> 
> 
> 
> If you need to do it from the repo, you can follow the steps [here](https://supabase.com/docs/guides/self-hosting/docker)

Thank you!

I'll try this and give you a feedback
